worlds/terraria/docs/setup_en.md

```diff
@@ -7,16 +7,28 @@ and [TModLoader](https://store.steampowered.com/app/1281930/tModLoader/) on Stea
 
 ## Installing the Archipelago Mod
 
-Subscribe to [the mod](https://steamcommunity.com/sharedfiles/filedetails/?id=2922217554) on Steam.
+1. Subscribe to [the mod](https://steamcommunity.com/sharedfiles/filedetails/?id=2922217554) on Steam
+2. Open TModLoader
+3. Go to `Workshop > Manage Mods` and enable the Archipelago mod
```

Comment:
Unless this is standard Archipelago style, this should use quotes, since it's prose. Backticks are for code formatting. So, "Workshop" > "Manage Mods"

Reply:
I will delegate this decision to @LegendaryLinux since he is the primary maintainer of the website.

Reply:
I believe the convention is to use backticks (code) to indicate either code directly, or information that can be found within a technical file. As this is an option within a game menu, I think quotes are appropriate here.